### ISSUE TITLE
Check status of cluster auto-scaling addon

### ISSUE THREAD

**Issue by leahnp** [samsung-cnct/kraken-lib#966]
Currently our default kraken config does not enable cluster auto-scaling due to concerns outlined here: https://github.com/samsung-cnct/kraken-lib/issues/963

This ticket is a research spike to evaluate the current state of the addon and decide if it is in a state that the team feels comfortable with making it enabled by default. 

Please outline and address each of the issues raised by @davidewatson in the initial #963 issue and any subsequent issues that were added.

**Comment by coffeepac** [comment]
output:  autoscaling is enabled by default || another issue with things to go fix

**Comment by coffeepac** [comment]
blocked by #963
